<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LiveTracker</class>
 <widget class="QMainWindow" name="LiveTracker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>343</width>
    <height>467</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>实时盯盘助手</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tblStockList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>321</width>
      <height>192</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>70</number>
    </attribute>
    <column>
     <property name="text">
      <string>股票代码</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>股票名称</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>当前价格</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>涨跌幅</string>
     </property>
    </column>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>321</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="btnAddStock">
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>添加</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRemoveStock">
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>删除</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClearStocks">
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>清空</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="grpEventSignal">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>321</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="title">
     <string>盘中异动提示</string>
    </property>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>301</width>
       <height>108</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QCheckBox" name="cbxDailyUpPercent">
          <property name="font">
           <font>
            <family>Microsoft YaHei UI</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>日内涨幅达到</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spbDailyUpPercent">
          <property name="font">
           <font>
            <family>Microsoft YaHei UI</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="suffix">
           <string>%</string>
          </property>
          <property name="maximum">
           <number>10</number>
          </property>
          <property name="value">
           <number>5</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cbxDailyDownPercent">
          <property name="font">
           <font>
            <family>Microsoft YaHei UI</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>跌幅达到</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spbDailyDownPercent">
          <property name="font">
           <font>
            <family>Microsoft YaHei UI</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="suffix">
           <string>%</string>
          </property>
          <property name="maximum">
           <number>10</number>
          </property>
          <property name="value">
           <number>5</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QCheckBox" name="cbxFiveMinUpPercent">
          <property name="font">
           <font>
            <family>Microsoft YaHei UI</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>5分钟涨幅达到</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spbFiveMinUpPercent">
          <property name="font">
           <font>
            <family>Microsoft YaHei UI</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="suffix">
           <string>%</string>
          </property>
          <property name="maximum">
           <number>10</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cbxFiveMinDownPercent">
          <property name="font">
           <font>
            <family>Microsoft YaHei UI</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>跌幅达到</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spbFiveMinDownPercent">
          <property name="font">
           <font>
            <family>Microsoft YaHei UI</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="suffix">
           <string>%</string>
          </property>
          <property name="maximum">
           <number>10</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QCheckBox" name="cbxSingleBuyAmount">
          <property name="font">
           <font>
            <family>Microsoft YaHei UI</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>单笔买单</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spbSingleBuyAmount">
          <property name="font">
           <font>
            <family>Microsoft YaHei UI</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="suffix">
           <string>万元</string>
          </property>
          <property name="minimum">
           <number>10</number>
          </property>
          <property name="maximum">
           <number>10000</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cbxSingleSellAmount">
          <property name="font">
           <font>
            <family>Microsoft YaHei UI</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>单笔卖单</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spbSingleSellAmount">
          <property name="font">
           <font>
            <family>Microsoft YaHei UI</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="suffix">
           <string>万元</string>
          </property>
          <property name="minimum">
           <number>10</number>
          </property>
          <property name="maximum">
           <number>10000</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QCheckBox" name="cbxActiveBuy">
          <property name="font">
           <font>
            <family>Microsoft YaHei UI</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>1分钟外盘大于</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spbActiveBuyPercent">
          <property name="font">
           <font>
            <family>Microsoft YaHei UI</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="suffix">
           <string>%</string>
          </property>
          <property name="minimum">
           <number>50</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>80</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="cbxActiveSell">
          <property name="font">
           <font>
            <family>Microsoft YaHei UI</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="text">
           <string>内盘大于</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spbActiveSellPercent">
          <property name="font">
           <font>
            <family>Microsoft YaHei UI</family>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="suffix">
           <string>%</string>
          </property>
          <property name="minimum">
           <number>50</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>80</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>321</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft YaHei UI</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnImportStockList">
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>导入股票列表</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnStartTracking">
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>开始盯盘</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnStopTracking">
       <property name="font">
        <font>
         <family>Microsoft YaHei UI</family>
         <pointsize>9</pointsize>
        </font>
       </property>
       <property name="text">
        <string>停止盯盘</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>343</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnImportStockList</sender>
   <signal>clicked()</signal>
   <receiver>LiveTracker</receiver>
   <slot>import_stock_list()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>83</x>
     <y>27</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnStartTracking</sender>
   <signal>clicked()</signal>
   <receiver>LiveTracker</receiver>
   <slot>start_monitoring()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>187</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>25</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnStopTracking</sender>
   <signal>clicked()</signal>
   <receiver>LiveTracker</receiver>
   <slot>stop_monitoring()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>305</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>310</x>
     <y>25</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnAddStock</sender>
   <signal>clicked()</signal>
   <receiver>LiveTracker</receiver>
   <slot>add_stock_code()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>271</y>
    </hint>
    <hint type="destinationlabel">
     <x>74</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnRemoveStock</sender>
   <signal>clicked()</signal>
   <receiver>LiveTracker</receiver>
   <slot>remove_stock_code()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>267</y>
    </hint>
    <hint type="destinationlabel">
     <x>198</x>
     <y>258</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnClearStocks</sender>
   <signal>clicked()</signal>
   <receiver>LiveTracker</receiver>
   <slot>clear_stock_list()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>313</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>328</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbxDailyUpPercent</sender>
   <signal>clicked(bool)</signal>
   <receiver>spbDailyUpPercent</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>335</y>
    </hint>
    <hint type="destinationlabel">
     <x>142</x>
     <y>335</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbxDailyDownPercent</sender>
   <signal>clicked(bool)</signal>
   <receiver>spbDailyDownPercent</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>332</y>
    </hint>
    <hint type="destinationlabel">
     <x>281</x>
     <y>334</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbxFiveMinUpPercent</sender>
   <signal>clicked(bool)</signal>
   <receiver>spbFiveMinUpPercent</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>105</x>
     <y>361</y>
    </hint>
    <hint type="destinationlabel">
     <x>140</x>
     <y>362</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbxFiveMinDownPercent</sender>
   <signal>clicked(bool)</signal>
   <receiver>spbFiveMinDownPercent</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>358</y>
    </hint>
    <hint type="destinationlabel">
     <x>276</x>
     <y>360</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbxSingleBuyAmount</sender>
   <signal>clicked(bool)</signal>
   <receiver>spbSingleBuyAmount</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>76</x>
     <y>387</y>
    </hint>
    <hint type="destinationlabel">
     <x>114</x>
     <y>389</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbxSingleSellAmount</sender>
   <signal>clicked(bool)</signal>
   <receiver>spbSingleSellAmount</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>388</y>
    </hint>
    <hint type="destinationlabel">
     <x>263</x>
     <y>388</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbxActiveBuy</sender>
   <signal>clicked(bool)</signal>
   <receiver>spbActiveBuyPercent</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>417</y>
    </hint>
    <hint type="destinationlabel">
     <x>138</x>
     <y>417</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cbxActiveSell</sender>
   <signal>clicked(bool)</signal>
   <receiver>spbActiveSellPercent</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>229</x>
     <y>415</y>
    </hint>
    <hint type="destinationlabel">
     <x>269</x>
     <y>415</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>import_stock_list()</slot>
  <slot>start_monitoring()</slot>
  <slot>stop_monitoring()</slot>
  <slot>add_stock_code()</slot>
  <slot>remove_stock_code()</slot>
  <slot>clear_stock_list()</slot>
 </slots>
</ui>
